ladybug/designday.py

```diff
@@ -866,13 +911,23 @@ def __repr__(self):
 class HumidityCondition(object):
     """Represents humidity conditions on the design day.
 
-    Properties:
+    Args:
         hum_type: Choose from
             Wetbulb, Dewpoint, HumidityRatio, Enthalpy
         hum_value: The value of the condition above
         barometric_pressure: Default is to use pressure at sea level
         schedule: Optional humidity schedule
         wet_bulb_range: Optional wet bulb temperature range
+
+    Properties:
+        * hum_type
+        * hum_value
+        * barometric_pressure
+        * schedule
+        * wet_bulb_range
+        * hourly_pressure
+        * humid_types
+        * isHumidityCondition
```

Comment:
Remove `isHumidityCondition` from this list since the whole property is going to be removed soon.